tests/random_test.py

```diff
@@ -539,10 +545,10 @@ def testPoisson(self, lam, dtype):
 
   def testPoissonBatched(self):
     key = random.PRNGKey(0)
-    lam = jnp.concatenate([2 * jnp.ones(10000), 20 * jnp.ones(10000)])
+    lam = jnp.concatenate([2 * jnp.ones(10000), 11 * jnp.ones(10000)])
```

Comment:
Any reason these are changed to 11? I would imagine the test should work with 20

Reply:
It fails with the combination of PRNG Key 0 and lam=20 because an unlikely value happens to be produced. It wasn't caught previously because we removed all values with counts less than 5.

Reply:
The docs seem to suggest that this can produce false negatives when counts are less than 5, but I don't know how to avoid that in long-tailed distributions while still maintaining the requirement that the expected and actual distribution add up to the same number.

Reply:
Was it that a rare value was produced, or just that the pmf has too much nonzero mass beyond ~100 when the parameter is 20?

Reply:
I don’t think it’s from the pig having power beyond the cutoff, because that would lead to a RuntimeError form scipy, where the failure here was a p value that was out of range.